ckanext/dia_theme/fanstatic/dia_custom.css

```diff
@@ -7876,6 +7876,10 @@ h4 small {
   height: 35px;
   background-position: -288px -62px;
 }
+.format-label[data-format=geojson],
```

Comment:
Change here for background pos override. lessc osx updates many of the hex strings.